bin/.travis/prepare_unittest.sh

```diff
@@ -12,6 +12,15 @@ if [ "$DB" = "postgresql" ] ; then psql -c "CREATE DATABASE $DB_NAME;" -U postgr
 echo "> Setup github auth key to not reach api limit"
 cp bin/.travis/composer-auth.json ~/.composer/auth.json
 
+# solr package search API integration tests
+if [ "$TEST_CONFIG" = "phpunit-integration-legacy-solr.xml" ] ; then
+    composer require -v --no-progress --no-interaction ezsystems/ezplatform-solr-search-engine:dev-master
```

Comment:
Cool, but to avoid issues with this doing composer install for us, can change this further to something like:

```bash
COMPOSER_UPDATE=""
# solr package search API integration tests
if [ "$TEST_CONFIG" = "phpunit-integration-legacy-solr.xml" ] ; then
    composer require --no-update ezsystems/ezplatform-solr-search-engine:dev-master
    COMPOSER_UPDATE="true"

    # required to satisfy the requirements of ezsystems/ezplatform-solr-search-engine:dev-master
    # (so that composer resolves ezpublish-kernel's self.version)
    git fetch --unshallow
fi

# Switch to another Symfony version if asked for
if [ "$SYMFONY_VERSION" != "" ] ; then
    composer require --no-update symfony/symfony="${SYMFONY_VERSION}"
    COMPOSER_UPDATE="true"
fi

# Install packages with composer update if asked for to make sure not use composer.lock if present
if [ "$COMPOSER_UPDATE" = "true" ] ; then
    echo "> Install dependencies through Composer (using update as other packages was requested)"
    composer update --no-progress --no-interaction --prefer-dist
else
    echo "> Install dependencies through Composer"
    composer install --no-progress --no-interaction --prefer-dist
fi
```